_posts/2022-09-05-edge-example-app-cpu-temp.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Edge Example App: CPU Temperature"
+date:   2022-09-05 03:27:05 +0100
+categories: flotta
+author: Ahmad Ateya <ahmad.m.ateya@gmail.com>
+tags:
+- flotta
+- devices
+- workloads
+
+summary-1:
+---
+Edge Example App is an app for Flotta Edge devices, with a workload that will be deployed on the device that has 2 main features:
+
+Sensing the Internet (which helps to construct devices network topology). (see [part 1]() and [part 2]())
```

Comment:
Do not forget: before merging the links should be filled

Reply:
fixed them [here](https://github.com/project-flotta/project-flotta.github.io/pull/113)